Ajustar Release

**Descrição do problema**
Arquivo `pyproject.toml` ajustá-lo

https://github.com/brazilian-utils/brutils-python/actions/runs/7423824289/job/20202106589

```
Run poetry build
  
Creating virtualenv brutils-UDYM-PsD-py3.10 in /home/runner/.cache/pypoetry/virtualenvs
Building brutils (2.0.0)
Using a plaintext file to store credentials

Publishing brutils (2.0.0) to PyPI
 - Uploading brutils-2.0.0-py3-none-any.whl 0%

 - Uploading brutils-2.0.0-py3-none-any.whl [10](https://github.com/brazilian-utils/brutils-python/actions/runs/7423824289/job/20202106589#step:4:11)0%
HTTP Error 400: File already exists. See https://pypi.org/help/#file-name-reuse for more information. | b'<html>\n <head>\n  <title>400 File already exists. See https://pypi.org/help/#file-name-reuse for more information.\n \n <body>\n  <h1>400 File already exists. See https://pypi.org/help/#file-name-reuse for more information.\n  The server could not comply with the request since it is either malformed or otherwise incorrect.<br/><br/>\nFile already exists. See https://pypi.org/help/#file-name-reuse for more information.\n\n\n \n'
Error: Process completed with exit code 1.
```

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "brutils"
-version = "2.0.0"
+version = "2.1.0"
 description = "Utils library for specific Brazilian businesses"
 authors = ["The Brazilian Utils Organization"]
 license = "MIT"

